src/index.js

```diff
@@ -1,10 +1,17 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import App from "./App";
+import { Provider } from "react-redux";
+import { BrowserRouter } from "react-router-dom";
+import store from './redux/store';
```

Comment:
- [ ] kindly the import Can't resolve the file. please fix it.

![revieweerro](https://user-images.githubusercontent.com/104437323/226542295-d4d47f59-ede6-478f-8620-653f04fd35c1.png)